frontend/talentsearch/src/js/lang/fr.json

```diff
@@ -1528,12 +1528,16 @@
     "defaultMessage": "Groupes visés par l’équité en matière d’emploi",
     "description": "Legend for the employment equity checklist"
   },
-  "1xRst5": {
-    "defaultMessage": "Résultats : <primary><testId>{candidateCount}</testId></primary> candidats correspondants à vos critères",
+   "Fjikio": {
+    "defaultMessage": "{candidateCount, plural, =0 {Résultats : <primary><testId>{candidateCount}</testId></primary> candidats correspondants à vos critères} =1 {Résultats : <primary><testId>{candidateCount}</testId></primary> candidat correspondants à vos critères} other {Résultats : <primary><testId>{candidateCount}</testId></primary> candidats correspondants à vos critères}}",
```

Comment:
I think you need to remove the `primary` tags from `7Eg+38` to match the English string.

Reply:
ugh, I'm terrible for the little things 
here https://github.com/GCTC-NTGC/gc-digital-talent/pull/4278/commits/26cbbce231b55b8283f5ba63fdfc31f35a0c1465